pdc_client/__init__.py

```diff
@@ -130,8 +130,10 @@ class to perform requests.
                         ssl_verify = False
                     elif not insecure and not ssl_verify:
                         ssl_verify = True
-            elif ssl_verify is True:
+            elif ca_cert_or_insecure is True:
                 ssl_verify = False
+            else:
+                ssl_verify = ca_cert_or_insecure
```

Comment:
@ycheng-aa ,There  are 2 questions in your comments.
1. If no ca_cert in client, client also use ssl_verify and can get the value from config file or not insecure.
2. In command line, it can't set insecure as false, only `True` can be set. (https://github.com/product-definition-center/pdc-client/blob/master/pdc_client/runner.py#L130)